tests/README.md

```diff
@@ -2,29 +2,27 @@
 
 ## Installation
 
-## Enable required repositories on CentOS Stream 9
+The integration tests use the RESTful API of [podman](https://podman.io/getting-started/installation) to isolate hirte
+and the agents on multiple, containerized nodes. Therefore, a working installation of podman is required. Please refer
+to [podman installation instructions](https://podman.io/getting-started/installation).
 
-Following repositories are required on CentOS Stream 9 for successful installation:
+All required python packages are listed in the [requirements.txt](./requirements.txt) and can be installed using `pip`:
 
-```shell
-dnf install -y dnf-plugin-config-manager
-dnf config-manager -y --set-enabled crb
-dnf install -y epel-release
+```bash
+pip install -r requirements.txt
```

Comment:
Hmm, personally I hate mixing pip packages with RPMs, it just causes too many issues, when latest packages in pip cannot be used with packages on your OS. That's why I used only RPMs to install it, because it's sure that those packages aligns fine with Fedora or CentOS Stream 9.

Reply:
I readded the RPM section, but I'd still keep the pip section due to two reasons: 
1. enable non-rpm based distros
2. using virtual envs for python dev is quite useful for isolation so that I can use different package versions etc. for other  projects

Reply:
+1